sonar-text-dotnet/src/SonarLint.Secrets.DotNet/SonarLint.Secrets.DotNet.csproj

```diff
@@ -14,11 +14,12 @@
 
   <PropertyGroup Label="Versioning - don't edit these values; edit the value in Directory.Build.props instead">
     <!-- The build number and commit id are set by the CI pipeline. -->
+    <BuildNumber>0</BuildNumber>
     <BranchName>not-set</BranchName>
     <CommitId>not-set</CommitId>
-    <AssemblyVersion>$(Version).$(BUILD_NUMBER)</AssemblyVersion>
-    <FileVersion>$(Version).$(BUILD_NUMBER)</FileVersion>
-    <InformationalVersion>$(Version).$(BUILD_NUMBER) Branch: $(CIRRUS_BRANCH) CommitId: $(CIRRUS_CHANGE_IN_REPO) Configuration: $(BUILD_CONFIGURATION)</InformationalVersion>
+    <AssemblyVersion>$(Version).$(BuildNumber)</AssemblyVersion>
+    <FileVersion>$(Version).$(BuildNumber)</FileVersion>
+    <InformationalVersion>$(Version).$(BuildNumber) Branch: $(BranchName) CommitId: $(CommitId) Configuration: $(BUILD_CONFIGURATION)</InformationalVersion>
```

Comment:
I think that in this case, we can use directly the `$(Configuration)` predefined MsBuild variable.